README.md

```diff
@@ -13,16 +13,36 @@ definitions within Vim.
 Usage
 -----
 
-While on a factory reference in a spec file, run `:Rfactory` to navigate to
-the factory definition. If the current line contains a FactoryGirl trait
-reference, you will be taken to that trait.
+With your cursor anywhere on a line containing a FactoryGirl reference, e.g.
+`create(:user)`, run `:Rfactory` to navigate to the factory definition. If the
```

Comment:
"navigate directly to the factory definition"? I thought it just went to the factory file, not `factory :user`.